enamel/main.py

```diff
@@ -66,10 +79,13 @@ def home():
 
 
 def server_boot():
+    if flask.request.content_type != 'application/json':
+        flask.abort(400)
```

Comment:
Thanks.  I agree that this is the wrong place to enforce it.  I'll add a comment noting that.